### PR TITLE
Support parsing speculation_store_nypass

### PR DESCRIPTION
**Pull request by ice1000**
@siddontang @overvenus 
Extracted from #3

### CODE REVIEW

File: src/pid/status.rs [hunk 1]
```diff
@@ -255,8 +259,9 @@ named!(parse_mems_allowed<Box<[u8]> >, delimited!(tag!("Mems_allowed:\t"), parse
 named!(parse_cpus_allowed_list<()>, chain!(tag!("Cpus_allowed_list:\t") ~ not_line_ending ~ line_ending, || { () }));
 named!(parse_mems_allowed_list<()>, chain!(tag!("Mems_allowed_list:\t") ~ not_line_ending ~ line_ending, || { () }));
 
-named!(parse_voluntary_ctxt_switches<u64>,    delimited!(tag!("voluntary_ctxt_switches:\t"),    parse_u64, line_ending));
-named!(parse_nonvoluntary_ctxt_switches<u64>, delimited!(tag!("nonvoluntary_ctxt_switches:\t"), parse_u64, line_ending));
+named!(parse_speculation_store_nypass<String>, delimited!(tag!("Speculation_Store_Bypass:\t"),   parse_line, line_ending));
```
Comment (overvenus) [inline src/pid/status.rs]: Should be `parse_speculation_store_bypass.
```suggestion
named!(parse_speculation_store_bypass<String>, delimited!(tag!("Speculation_Store_Bypass:\t"),   parse_line, line_ending));
```

File: src/pid/status.rs [hunk 2]
```diff
@@ -175,6 +178,7 @@ pub struct Status {
     pub voluntary_ctxt_switches: u64,
     /// Number of involuntary context switches.
     pub nonvoluntary_ctxt_switches: u64,
+    pub speculation_store_nypass: String,
```
Comment (overvenus) [inline src/pid/status.rs]:
```suggestion
    pub speculation_store_bypass: String,
```

File: src/pid/status.rs (hunk 1 above)
Comment (ice1000) [reply src/pid/status.rs]: Address comment

**Comment by ice1000** [comment]
Wat

**Pull request merged**